openlibrary/templates/books/custom_carousel.html

```diff
@@ -44,7 +44,7 @@
           $else:
             src="$(cover_url)"/>
         $else:
-          <div class="carousel__item__blankcover">
+          <div class="carousel__item__blankcover"  title="$(title)">
```

Comment:
This fixes `title` not being present for initially loaded books that are missing cover images.  Good catch!